tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -287,6 +299,28 @@ impl<F, T> Format<F, T> {
             ..self
         }
     }
+
+    /// Sets whether or not the [thread ID] of the current thread is displayed
+    /// when formatting events
+    ///
+    /// [thread ID]: https://doc.rust-lang.org/stable/std/thread/struct.ThreadId.html
+    pub fn with_thread_id(self, display_thread_id: bool) -> Format<F, T> {
```

Comment:
Should these fn names also be plural here? My reason for keeping them singular is that they deal with a single event being part of `Format`. WDYT?

Reply:
IMO, I think it's better for them to be consistent --- it's annoying to call a method on one type, and then try to call it on a different type, and get an error because the name is slightly different. Also, the formatter will almost certainly be used to format more than one event, too.

But, I could definitely be convinced otherwise!